tests/bricks/test_conv.py

```diff
@@ -195,19 +195,58 @@ def test_no_input_size():
                          c.initialize)
 
 
-def test_tied_biases():
+def test_untied_biases():
     x = tensor.tensor4('x')
     num_channels = 4
     num_filters = 3
     batch_size = 5
     filter_size = (3, 3)
     conv = Convolutional(filter_size, num_filters, num_channels,
                          weights_init=Constant(1.), biases_init=Constant(2.),
-                         tied_biases=True)
+                         image_size = (28, 30), tied_biases=False)
```

Comment:
Think there's a PEP8 issue there (spaces around the equals sign). We supposedly have a linter to catch this sort of stuff but it doesn't appear to be working.